DigitalLearningSolutions.Data/Services/PostLearningAssessmentService.cs

```diff
@@ -66,5 +65,58 @@ GROUP BY
                 new { customisationId, candidateId, sectionId }
             );
         }
+
+        public PostLearningContent? GetPostLearningContent(int customisationId, int sectionId)
+        {
+            PostLearningContent? postLearningContent = null;
+            return connection.Query<PostLearningContent, int, PostLearningContent>(
+                @"
+                    SELECT
+                        Applications.ApplicationName,
+                        Customisations.CustomisationName,
+                        Sections.SectionName,
+                        Sections.PLAssessPath,
+                        Applications.PLAPassThreshold,
+                        Customisations.CurrentVersion,
+                        CASE WHEN Tutorials.OriginalTutorialID > 0
+                            THEN Tutorials.OriginalTutorialID
+                            ELSE Tutorials.TutorialID
+                        END AS id
+                    FROM Sections
+                        INNER JOIN Customisations
+                            ON Customisations.ApplicationID = Sections.ApplicationID
+                            AND Customisations.Active = 1
+                            AND Customisations.IsAssessed = 1
+                        INNER JOIN Applications
+                            ON Applications.ApplicationID = Sections.ApplicationID
+                        INNER JOIN CustomisationTutorials
+                            ON CustomisationTutorials.CustomisationID = Customisations.CustomisationID
+                        INNER JOIN Tutorials
+                            ON Tutorials.TutorialID = CustomisationTutorials.TutorialID
+                            AND Tutorials.SectionID = Sections.SectionID
+                            AND Tutorials.ArchivedDate IS NULL
+                    WHERE
+                        Customisations.CustomisationID = @customisationId
+                        AND Sections.SectionID = @sectionId
+                        AND Sections.ArchivedDate IS NULL
+                        AND Sections.PLAssessPath IS NOT NULL
+                    ORDER BY
+                        Tutorials.OrderByNumber,
+                        Tutorials.TutorialID",
+                (postLearning, tutorialId) =>
+                {
+                    if (postLearningContent == null)
```

Comment:
Can simplify with `??=` 🙂

Reply:
We seem to be using the current notation for most/all of our other queries too, should I change those too?

Reply:
I believe most of the queries have more complex logic, they're doing more than just assigning (e.g. adding to scores). If you can find any which are just doing this then feel free to change them (although just for the learning menu to avoid conflicts)🙂